### PR TITLE
Add v5 PV 2 docs for iOS and Android

### PR DESCRIPTION
**Pull request by Sporiff**
Closes https://adjustcom.atlassian.net/browse/ADAPP-18204

### CODE REVIEW

**Comment by Sporiff** [comment]
@shashanksu @Aditi3 I've added you as reviewers just to check everything looks right from an SDK perspective. Here are the direct links:

- [Android](https://dev-docs-git-adapp-18204-documentation-p-2966d1-product-content.vercel.app/en/sdk/android/features/events?version=v5#purchase-verification)
- [iOS](https://dev-docs-git-adapp-18204-documentation-p-2966d1-product-content.vercel.app/en/sdk/ios/features/events?version=v5#purchase-verification)

File: src/content/docs/sdk/ios/features/events.mdx [hunk 1]
```diff
@@ -282,41 +282,134 @@ Parameters:
...
 <Tabs>
 <Tab title="Swift" sync="swift">
 
 ```swift
-let receiptURL = Bundle.main.appStoreReceiptURL;
-let receipt = try Data(contentsOf: receiptURL, options: .alwaysMapped)
+let event = ADJEvent(eventToken: "g3mfiw")!
+event.setProductId("product-id")
+event.setTransactionId("transaction-id")
+event.setRevenue(0.01, currency: "EUR")
```
Comment (Aditi3) [inline src/content/docs/sdk/ios/features/events.mdx:342]:
```suggestion
event.setReceipt(yourReceiptData)
```

File: src/content/docs/sdk/ios/features/events.mdx [hunk 2]
```diff
@@ -282,41 +282,134 @@ Parameters:
...
+}
+```
+
+</Tab>
+<Tab title="Objective-C" sync="objc">
+
+```objc
+ADJAppStorePurchase *appStorePurchase = [[ADJAppStorePurchase alloc] initWithTransactionId:yourTranscationId
+                                                                              productId:yourProductId
+                                                                              andReceipt:yourReceiptData];
```
Comment (Aditi3) [inline src/content/docs/sdk/ios/features/events.mdx:408]:
```suggestion
ADJAppStorePurchase *appStorePurchase = [[ADJAppStorePurchase alloc] 
                                                                               initWithTransactionId:yourTranscationId
                                                                               productId:yourProductId
                                                                               andReceipt:yourReceiptData];
```

**Review by Aditi3 — APPROVED** [review]
I provided one extra parameter, which was not needed, and updated it with the right one, the rest looks good.

**Comment by shashanksu** [comment]
@Sporiff Android part looks good to me.

**Comment by Sporiff** [comment]
Following up from @uerceg 's comments on [ADAPP-18204](https://adjustcom.atlassian.net/browse/ADAPP-18204), I've moved the documentation for PV to a separate page for each platform. I've also updated the code snippets in line with the suggestion provided, and added the Unity documentation.

@Aditi3 @shashanksu could you just take another look at the updated code snippets? @uerceg Could you verify Unity?

- [Android](https://dev-docs-git-adapp-18204-documentation-p-2966d1-product-content.vercel.app/en/sdk/android/features/purchase-verification?version=v5)
- [iOS](https://dev-docs-git-adapp-18204-documentation-p-2966d1-product-content.vercel.app/en/sdk/ios/features/purchase-verification?version=v5)
- [Unity](https://dev-docs-git-adapp-18204-documentation-p-2966d1-product-content.vercel.app/en/sdk/unity/features/purchase-verification?version=v5)

**Review by Aditi3 — APPROVED** [review]
iOS part looks good! 🚀

**Comment by shashanksu** [comment]
Android part looks good too!

**Pull request merged**